lpa-submissions-e2e-tests/cypress/support/common/verifyPageHeading.js

```diff
@@ -0,0 +1,10 @@
+/// <reference types = "Cypress"/>
+import { pageHeading } from '../PageObjects/common-page-objects';
+
+module.exports = (pageHeadingId) => {
```

Comment:
Why is this labelled as an ID? From the function it looks like the text gets passed instead.

Reply:
Done.